tests/test_generate_summary_outputs.py

```diff
@@ -102,7 +102,7 @@ def test_calculate_gene_rank_difference(self):
                     "Gene": "GCDH",
                     "/path/to/results_directory1": 1,
                     "/path/to/results_directory2": 5,
-                    "rank_decrease": 4,
+                    "rank_change": 4,
```

Comment:
Isn't a rank decrease of 4 equal to a rank change of -4?

Reply:
If we are comparing `results_directory1` to `results_directory2` using `results_directory1` as the baseline, hasn't the rank increased by 4 ranks?

Reply:
Changed now!